pyproject.toml

```diff
@@ -55,6 +56,7 @@ myst-parser = {version = ">=0.16.1"}
 
 [tool.poetry.scripts]
 camelot = "camelot.__main__:main"
+table_extraction = "pypdf_table_extraction.__main__:main"
```

Comment:
probably better to do `pdf_table_extraction` or `pypdf_table_extraction`

Reply:
Should we go with `pypdf_table_extraction`?

Reply:
AFAIK using `pypdf_table_extraction` would allow to use the same name for both calling the script directly and calling the module through `python -m pypdf_table_extraction`, which would avoid confusion.

Reply:
I agree, calling the script with the same name as for the module is the way to go 


There is already another module named pdf_table_extraction so using that name would also lead to confusion.

Reply:
Added a commit to make it `pypdf_table_extraction`